internal/ingress/controller/template/configmap.go

```diff
@@ -150,6 +152,25 @@ func ReadConfig(src map[string]string) config.Configuration {
 		}
 	}
 
+	// Verify that the configured global auth is parsable as URL. if not, set the default value
+	if val, ok := conf[globalAuthURL]; ok {
+		delete(conf, globalAuthURL)
+
+		authURL, err := url.Parse(val)
+		if err != nil {
+			glog.Warningf("Global auth location denied, reason: url is not")
```

Comment:
The log message seems incomplete. Also can you include the `err` in the message as well?

Reply:
I assume it should be ".. is not valid"))
Yes, will append the `err` to the end.